specification/network/resource-manager/Microsoft.Network/stable/2022-05-01/virtualWan.json

```diff
@@ -5683,6 +5917,126 @@
         }
       }
     },
+    "subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualHubs/{virtualHubName}/inboundRoutes": {
```

Comment:
For inboundRoutes and outboundRoutes, do we first need implement it in NRP and nfv?

Reply:
Its already implemented in nrp and nfv

Reply:
done